### ISSUE TITLE
WebSocketException: Unable to connect to the remote server - Android

### ISSUE THREAD

**Issue by LukaszwBeamable** [reown-com/reown-dotnet#19]
Hi i tried to build Unity project with Reown for Android and i'm blocked with WebSocketException issue, which occures after i chose wallet type from modal view (ofc. appKit initialization was sucesfull without any error etc.). 

 **Fun fact: for webgl everyting is fine, i can connect without anny errors etc - all reown features works as expected.**
 
Logs:
[reown_android_logs.txt](https://github.com/user-attachments/files/17844010/reown_android_logs.txt)
Screens:
![Screenshot_20241121-111712](https://github.com/user-attachments/assets/68f82f31-b0cf-449f-a0af-ba68a6b88058)
![Screenshot_20241121-111717](https://github.com/user-attachments/assets/bb7e0b99-2e03-4401-876d-73d9e642ad5b)

Do i need to add any extra Android Manifest setup to make it works, or problem is related to something completely different?

My Unity version is 2022.3.50f1

**Comment by skibitsky** [comment]
Hello @LukaszwBeamable,

Have you verified any domain at cloud.reown.com? Do you experience issues only on Android device, or also in the Unity Editor with Android set as the target platform?

**Comment by LukaszwBeamable** [comment]
@skibitsky Yes i have domain verifired, and i added Android Package Name in Reown Dashboard. In the Unity Editor with Android set as the target platform i can see fully initialized modals but i can only copy a link, hovewer i have an Argument Exception error with empty url when i try to do this:

![image](https://github.com/user-attachments/assets/742c9546-ac83-4cdf-8b53-361d9586d6f5)

**Comment by skibitsky** [comment]
Thanks, this will be fixed in the next release today or tomorrow. Temporary workaround: remove **domain verification** in the cloud. Apologies for the inconvenience 🙏

**Comment by skibitsky** [comment]
Hello @LukaszwBeamable,

A potential solution was implemented in version 1.1.0. Could you please update to this version and verify whether the issue still occurs?

**Comment by skibitsky** [comment]
Could you please try v1.1.1?

**Comment by talha-trili** [comment]
So that issue is gone but now i get 2 new ones unfortunately.
My dependencies
```
"dependencies"    : {
	"com.unity.nuget.newtonsoft-json": "3.2.1",
	"com.nethereum.unity"            : "4.26.0",
	"com.reown.core"                 : "1.1.0",
	"com.reown.core.crypto"          : "1.1.0",
	"com.reown.core.common"          : "1.1.0",
	"com.reown.core.network"         : "1.1.0",
	"com.reown.core.storage"         : "1.1.0",
	"com.reown.sign"                 : "1.1.0",
	"com.reown.sign.unity"           : "1.1.0",
	"com.reown.sign.nethereum"       : "1.1.0",
	"com.reown.sign.nethereum.unity" : "1.1.0",
	"com.reown.appkit.unity"         : "1.1.1"
  },
```
And it doesnt matter if i update all dependencies to 1.1.1

If i switch back to 1.0.1 it works so below versions are fine.

```
"dependencies"    : {
	"com.unity.nuget.newtonsoft-json": "3.2.1",
	"com.nethereum.unity"            : "4.26.0",
	"com.reown.core"                 : "1.1.0",
	"com.reown.core.crypto"          : "1.1.0",
	"com.reown.core.common"          : "1.1.0",
	"com.reown.core.network"         : "1.1.0",
	"com.reown.core.storage"         : "1.1.0",
	"com.reown.sign"                 : "1.1.0",
	"com.reown.sign.unity"           : "1.1.0",
	"com.reown.sign.nethereum"       : "1.1.0",
	"com.reown.sign.nethereum.unity" : "1.1.0",
	"com.reown.appkit.unity"         : "1.0.1"
  },
```
Error screenshots.
<img width="494" alt="image" src="https://github.com/user-attachments/assets/8f612fc1-28e0-4cfe-877f-ea50d33fd231">
<img width="491" alt="image" src="https://github.com/user-attachments/assets/79215c3a-63b7-4af6-99ec-174ac3348649">

**Comment by skibitsky** [comment]
The [com.reown.unity.dependencies](https://openupm.com/packages/com.reown.unity.dependencies/) package is missing from your list of dependencies. Previously, there was an issue with the OpenUPM CLI that prevented all dependency packages from being installed. However, this issue was resolved a few months ago, as far as I recall.

Could you please try to update OpenUPM CLI and reinstall AppKit package?
```bash
npm i -g openupm-cli@latest
openupm remove com.reown.appkit.unity
openupm add com.reown.appkit.unity
```

If it doesn't help, you can try to install missing dependency manually? 
```bash
openupm add com.reown.unity.dependencies
```

**Comment by LukaszwBeamable** [comment]
@skibitsky Unfortunately looks like that updating SDK breaks previously working feature on WebGL (for Browser & Unity Editor too). In editor QR code and link are not generated at all (previously it worked), and in bowser i have js errors:

Editror:
<img width="287" alt="image" src="https://github.com/user-attachments/assets/4b925dea-48d2-436e-bfe4-dd558d918e3e">

Bowser:
<img width="772" alt="image" src="https://github.com/user-attachments/assets/a381c313-b8b0-4453-a519-021867213a33">

**Comment by skibitsky** [comment]
@LukaszwBeamable that was my mistake, sorry 🙏 

I pushed a fix for `com.reown.unity.dependencies` and it's now available on [OpenUPM](https://openupm.com/packages/com.reown.unity.dependencies/?subPage=pipelines).

Could you please share you project id so I could look deeper into your issue?

**Comment by LukaszwBeamable** [comment]
> @LukaszwBeamable that was my mistake, sorry 🙏
> 
> I pushed a fix for `com.reown.unity.dependencies` and it's now available on [OpenUPM](https://openupm.com/packages/com.reown.unity.dependencies/?subPage=pipelines).
> 
> Could you please share you project id so I could look deeper into your issue?

`27b6287f691898e2dd836dace4909784`

**Comment by skibitsky** [comment]
Indeed, there is an issue with AppKit in the Unity Editor when the target platform is set to WebGL. I’ll address this in the next patch. As a temporary workaround, you can switch to another platform to use AppKit in the Editor.

Regarding WebGL in the browser, I’ve been unable to reproduce the issue so far. Could you please describe the exact steps you followed and specify which wallet you used to connect? If you are using a browser extension, kindly include the browser name as well 🙏

**Comment by skibitsky** [comment]
Hey @LukaszwBeamable, 

Could you please confirm if the issue was fixed for you in [v1.1.2](https://github.com/reown-com/reown-dotnet/releases/tag/com.reown.appkit.unity%2F1.1.2)?

**Comment by RambunctiousCookie** [comment]
Hi, I am on 1.1.2 and the `_connectionProposal.Uri` never resolves. Platform is on Android (Unity native client).

When using 

I think my dependencies are correct?
		com.reown.unity.dependencies@1.1.2
		com.reown.appkit.unity@1.1.2
		com.reown.core.common@1.1.2
		com.reown.core.crypto@1.1.2
		com.reown.core.network@1.1.2
		com.reown.core.storage@1.1.2
		com.reown.core@1.1.2
		com.reown.sign.nethereum.unity@1.1.2
		com.reown.sign.nethereum@1.1.2
		com.reown.sign.unity@1.1.2
		com.reown.sign@1.1.2
		
I can connect to wallet apps when using the `Reown.Customization.Unity` example in the `reown-dotnet` folder, but otherwise there is no way to connect

**Comment by RambunctiousCookie** [comment]
> Hey @RambunctiousCookie,
> 
> Could you please open a new issue and include your code snippet and project id?

No worries, there is no issue- I will use v1.0.1 since v1.1.2 is bugged

**Comment by LukaszwBeamable** [comment]
> Hey @LukaszwBeamable,
> 
> Could you please confirm if the issue was fixed for you in [v1.1.2](https://github.com/reown-com/reown-dotnet/releases/tag/com.reown.appkit.unity%2F1.1.2)?

For me it's fixed.
